redbeat/schedulers.py

```diff
@@ -191,11 +191,13 @@ def __init__(self, app, **kwargs):
     def setup_schedule(self):
         # cleanup old static entries
         client = redis(self.app)
-        previous = client.smembers(self.app.conf.REDBEAT_STATICS_KEY)
-        current = set(self.app.conf.CELERYBEAT_SCHEDULE.keys())
-        removed = previous - current
+        previous = set(key.decode('utf-8')
```

Comment:
Well, whaddayaknow, support for `decode_responses` was just merged into `fakeredis`. Still, I think it#s better, do decode explicitly.